test/integration/testLargeResultSet.js

```diff
@@ -269,7 +269,6 @@ describe('SNOW-743920: Large result set with ~35 chunks', function () {
     await testUtil.executeCmdAsync(connection, 'alter session set USE_CACHED_RESULT = false;');
     await testUtil.executeCmdAsync(connection, createTable);
     await testUtil.executeCmdAsync(connection, populateData);
-    configureLogger('TRACE');
```

Comment:
if you remove setting the TRACE level let's remove below setting the ERROR level

Reply:
ok